pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md

```diff
@@ -28,6 +28,22 @@ When testing your bot and you get an error in the response, you will be able to
 ## Dialog Flow V2 Invalid Action Body
 ## Description
 This error happens when the bot has an invalid body and the Google Dialog Flow V2 is unable to parse it. It can be a wrong parameter, a value, or a missing parameter. Even though it is shown as an error in the Third Party Bots, the bot gently ignores unknown actions and the conversation will continue.
+
+## Transfer Action Error
+## Description
+For this action the required parameter is skill that has to be a string, on the other hand agentId is an optional parameter that also has to be a string value.
+
+## Change TTR Action Error 
+## Description
+For this action the only required value is ttrType, it has to be a string.
+
+## Close Conversation Action Error
+## Description
+For this action there are no required parameters. In case the withoutPcs parameter is in the vendor response it has to be a boolean value. 
+
+## Faas Invocation Action Error
+## Description
+For this action the required values are lambdaUuid as string  and payload as object. FailOnError is an optional boolean value. 
```

Comment:
Please follow the format I provided above errors.

Reply:
done